### PR TITLE
Breadcrumb atom

### PR DESCRIPTION
**Pull request by 0c370t**

### CODE REVIEW

File: clients/web/src/lib/components/atoms/Breadcrumb/Breadcrumb.story.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Breadcrumbs
+
+The breadcrumb component is used to show the user where they are in the app.
```
Comment (zachstence) [inline clients/web/src/lib/components/atoms/Breadcrumb/Breadcrumb.story.md:3]: This is cute

File: clients/web/src/lib/components/atoms/Breadcrumb/addCrumb.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { getContext, setContext } from "svelte";
+import { type ICrumb, type BreadcrumbContext, BREADCRUMB_CONTEXT_KEY } from "./types";
+
+export const addCrumb = (crumb: ICrumb) => setContext<BreadcrumbContext>(BREADCRUMB_CONTEXT_KEY, [...getContext<BreadcrumbContext>(BREADCRUMB_CONTEXT_KEY), crumb])
```
Comment (zachstence) [inline clients/web/src/lib/components/atoms/Breadcrumb/addCrumb.ts:4]: I noticed inside the `Breadcrumb` component, we're using `hasContext` before `getContext`. Is that necessary here? Or is it not necessary in the other place?

File: clients/web/src/lib/components/atoms/Modal/Modal.story.svelte [hunk 3]
```diff
@@ -8,7 +8,7 @@
 </script>
 
 <Hst.Story title="Atoms/Modal" layout={{type: "grid", width: 800}}>
-	<Hst.Variant title="Default" {open}>
+	<Hst.Variant title="Default">
```
Comment (zachstence) [inline clients/web/src/lib/components/atoms/Modal/Modal.story.svelte:11]: Why remove this? Won't this prevent the story controls from working?

**Review by zachstence — CHANGES_REQUESTED** [review]

File: clients/web/src/lib/components/atoms/Breadcrumb/addCrumb.ts (hunk 2 above)
Comment (0c370t) [reply clients/web/src/lib/components/atoms/Breadcrumb/addCrumb.ts:4]: Good catch, it should be used here

File: clients/web/src/lib/components/atoms/Modal/Modal.story.svelte (hunk 3 above)
Comment (0c370t) [reply clients/web/src/lib/components/atoms/Modal/Modal.story.svelte:11]: There aren't any controls for this story; functionality still works.
This was creating type errors in vscode, hence the removal

**Review by zachstence — APPROVED** [review]

**Pull request merged**